tracker/event_tracker.go

```diff
@@ -101,6 +101,7 @@ func (e *EventTracker) Start(ctx context.Context) error {
 	go func() {
 		if err := tt.Sync(ctx); err != nil {
 			e.logger.Error("failed to sync", "error", err)
+			panic(err)
```

Comment:
@igorcrevar Added this to show it occurs in the tests as they currently exist. My earlier changes with the wait also uncovered this. Have tested this locally, expecting it to happen in PR also.

Is this intentional / expected behaviour? If so, I will just add wait for the other goroutine which is what I require

Reply:
Same thing for
```
		if err := blockTracker.Init(); err != nil {
			e.logger.Error("failed to init blocktracker", "error", err)

			panic(err)
		}
```

Reply:
Potentially these errors are unhandled for the sake of these tests that are failing, which will cause issues for people like myself. This config causes the error
```
			JSONRPCEndpoint: "testEndpoint",
```

Reply:
> Did you measure how much time takes for block tracker initialization/start and for initial batch sync?
No I haven't and I don't know the significance of those timings

Reply:
Maybe it would be good to know for how much time the main server routine gets blocked before initialization of event tracker finishes.

Reply:
I'm seeing `func (e *EventTracker) Start(ctx context.Context) error` take <20ms on my laptop. Those goroutines seemed unnecessary in the first place b/c the tracker and block tracker funcs were mainly just launching other goroutines themselves